### PR TITLE
Test to_dict with missing enum values

### PR DESCRIPTION
**Pull request by michael-sayapin**
In reference of https://github.com/danielgtaylor/python-betterproto/issues/93

### CODE REVIEW

**Comment by boukeversteegh** [comment]
Awesome, thanks a lot!
Could you please also run `black` on the test file (it’s an auto formatter), then we can merge this and someone can fix the bug later.

**Comment by michael-sayapin** [comment]
Done @boukeversteegh

File: betterproto/tests/inputs/to_dict_with_missing_enum/test_to_dict_with_missing_enum.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from betterproto.tests.output_betterproto.to_dict_with_missing_enum.to_dict_with_missing_enum import (
+    Test,
+    TestMyEnum,
+)
+
+
+def test_message_attributes():
```
Comment (boukeversteegh) [inline betterproto/tests/inputs/to_dict_with_missing_enum/test_to_dict_with_missing_enum.py:7]:
```suggestion
@pytest.mark.xfail(“#93”)
def test_message_attributes():
```

File: betterproto/tests/inputs/to_dict_with_missing_enum/test_to_dict_with_missing_enum.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from betterproto.tests.output_betterproto.to_dict_with_missing_enum.to_dict_with_missing_enum import (
+    Test,
+    TestMyEnum,
+)
+
```
Comment (boukeversteegh) [inline betterproto/tests/inputs/to_dict_with_missing_enum/test_to_dict_with_missing_enum.py:5]:
```suggestion
import pytest

```

File: betterproto/tests/inputs/to_dict_with_missing_enum/test_to_dict_with_missing_enum.py (hunk 1 above)
Comment (boukeversteegh) [reply betterproto/tests/inputs/to_dict_with_missing_enum/test_to_dict_with_missing_enum.py:7]: this suggested change (combined with the one below) is to make the tests pass.
sorry, i had not realized this was also still necessary for a merge

**Comment by boukeversteegh** [comment]
Closing in favor of fix PR #102

**Comment by boukeversteegh** [comment]
Thank you for setting up the initial test-case! Its now part of the above PR 🐍👍

**Pull request closed without merging**